src/content/packages-and-plugins/developing-packages.md

```diff
@@ -781,15 +781,13 @@ The native build systems that are invoked by FFI
 
 ### Step 3: Binding to native code
 
-To use the native code, bindings in Dart are needed.
+To use native code, you need Dart bindings. Instead of writing these bindings
+manually, generate them from the header file `src/hello.h` using the
+[`ffigen` package][]. To install `ffigen` and its dependencies, see the 
+[`ffigen` documentation][].
 
-To avoid writing these by hand,
-they are generated from the header file
-(`src/hello.h`) by [`package:ffigen`][].
-Reference the [ffigen docs][] for information
-on how to install this package.
 
-Regenerate the bindings by running the following:
+To regenerate the bindings, running the following command:
```

Comment:
```suggestion
To regenerate the bindings, run the following command:
```

Reply:
I saw this comment after I merged. Sorry about that.